perception/traffic_light_multi_camera_fusion/include/traffic_light_multi_camera_fusion/node.hpp

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/*
+ * Copyright 2023 Autoware Foundation. All rights reserved.
```

Comment:
I believe the headers use // rather than /* */
Additionally, is there a guideline when adding authors in the headers ?
I have seen only some packages in universe doing this

Reply:
@knzo25 
That's a good point.
I checked the files and found basically old statements (earlier than 2019) uses /**/ and newer statements uses //.
So I modified them.
About the authors, also only old code has authors so I removed the authors in traffic light packages.

Reply:
Thanks !